### DISCUSSION TITLE
Controller sending FUNC_ID_SERIAL_API_STARTED with Software reset?

### DISCUSSION THREAD

**Discussion by olavt** [zwave-js/zwave-js#5642] [General]
Have you seen this issue with UZB7 running firmware 7.19.1 and 7.19.2?

https://github.com/SiliconLabs/gecko_sdk/issues/40

**Comment by AlCalzone** [comment]
I think there was one report before.

**Reply by olavt** [reply]
Do you have a link or an idea on how to find it?

**Reply by AlCalzone** [reply]
Unfortunately not, thats somewhere  deep in a troubleshooting thread for a different issue.
